code/datums/emotes.dm

```diff
@@ -0,0 +1,353 @@
+
+/datum/emote/flutter
+	id = "flutter"
+	message_targeted = "#SOURCE flutters towards #TARGET!"
+	message_untargeted = "#SOURCE flutters."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/growl
+	id = "growl"
+	message_targeted = "#SOURCE growls at #TARGET!"
+	message_untargeted = "#SOURCE growls."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/stare
+	id = "stare"
+	message_targeted = "#SOURCE stares at #TARGET."
+	message_untargeted = "#SOURCE stares."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/watch
+	id = "watch"
+	message_targeted = "#SOURCE watches #TARGET closely."
+	message_untargeted = "#SOURCE watches."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/examine
+	id = "examine"
+	message_targeted = "#SOURCE examines #TARGET."
+	message_untargeted = "#SOURCE examines their surroundings."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/wince
+	id = "wince"
+	message_targeted = "#SOURCE winces at #TARGET."
+	message_untargeted = "#SOURCE winces."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/wince_heavy
+	id = "wince_heavy"
+	message_targeted = "#SOURCE winces painfully at #TARGET!"
+	message_untargeted = "#SOURCE winces painfully!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/blink
+	id = "blink"
+	message_targeted = "#SOURCE blinks at #TARGET."
+	message_untargeted = "#SOURCE blinks."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/blink_rapid
+	id = "blink_rapid"
+	message_targeted = "#SOURCE rapidly blinks at #TARGET."
+	message_untargeted = "#SOURCE rapidly blinks."
+	display_flags = EMOTE_VISIBLE
+
+/datum/emote/cough
+	id = "cough"
+	message_targeted = "#SOURCE coughs on #TARGET."
+	message_untargeted = "#SOURCE coughs."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_DISTANCE
+
+/datum/emote/sneeze
+	id = "sneeze"
+	message_targeted = "#SOURCE sneezes on #TARGET."
+	message_untargeted = "#SOURCE sneezes."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_DISTANCE
+
+/datum/emote/sniff
+	id = "sniff"
+	message_targeted = "#SOURCE sniffs #TARGET."
+	message_untargeted = "#SOURCE sniffs."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE | EMOTE_DISALLOW_DISTANCE
+
+/datum/emote/heave
+	id = "heave"
+	message_targeted = "#SOURCE dry heaves!"
+	message_untargeted = "#SOURCE dry heaves!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/sputter
+	id = "sputter"
+	message_targeted = "#SOURCE sputters!"
+	message_untargeted = "#SOURCE sputters!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/moan
+	id = "moan"
+	message_targeted = "#SOURCE moans."
+	message_untargeted = "#SOURCE moans."
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/moan_heavy
+	id = "moan_heavy"
+	message_targeted = "#SOURCE moans in pain!"
+	message_untargeted = "#SOURCE moans in pain!"
+	display_flags = EMOTE_AUDIBLE | EMOTE_VISIBLE
+
+/datum/emote/shiver
+	id = "shiver"
+	message_targeted = "#SOURCE shivers at #TARGET."
```

Comment:
This would be an interesting thing to see in real life. How do you shiver _at_ someone?

Reply:
Lean at them.